src/rnpkeys/rnpkeys.1.adoc

```diff
@@ -251,6 +251,10 @@ Since version 0.16.0 of RNP generated secret key is stored with bits set to a ne
 however, this may be needed to fix older keys or keys generated by other implementations.
 _KEY_ must specify the exact subkey via keyid or fingerprint.
 
+*--set-expire* _TIME_:::
+Set key expiration time. See the description of the *--expiration* option for possible time formats.
+Setting argument to 0 removes key expiration.
```

Comment:
For me seems more natural here something like '0 value means that key would never expire (which is not recommended due to security reasons)'.